src/algos/dynamic_programming/in_class/notes.MD

```diff
@@ -0,0 +1 @@
+implementation w/o priority queue is O(V^2+E). This makes it preferable when there are more nodes than edges.
```

Comment:
It's preferable when there are a lot of edges (e.g. a fully connected graph), because in that case, E ~ V^2.  So V^2 + E will be better than E log E in that case.